babi/main.py

```diff
@@ -124,6 +125,9 @@ def _filenames(filenames: List[str]) -> Tuple[List[Optional[str]], List[int]]:
 def main(argv: Optional[Sequence[str]] = None) -> int:
     parser = argparse.ArgumentParser()
     parser.add_argument('filenames', metavar='filename', nargs='*')
+    parser.add_argument(
+        '-T, --tabsize', default=4, type=int, metavar='cols', dest='tab_size',
+    )
```

Comment:
let's not add an option, I'd like to do this differently later and not have to break compat for this